docs/Help.md

```diff
@@ -120,3 +120,8 @@ Nightly ignores version checking and it will not attempt to update on its own. B
 
 ##### Issues
 Since nightly is a testing release, it may be a little unstable. If you find any bugs, make sure to report an issue on [GitHub](https://github.com/Chatterino/chatterino2/issues/new/choose). Before opening an issue make sure you're on latest nightly (to check it, go to Settings > About and compare commit hash) and that there's no duplicate issue already open.
+
+#### I see white borders on full screen
+This is a Windows issue it can be mitigated by turning on Compatibility mode for Chatterino. Right click `chatterino.exe` -> Properties -> Compatibility -> Enable "Run this program in compatibility mode for:" -> Select Windows 8 in the drop-down menu. 
+[Like this](images/help/boarders.png)
```

Comment:
```suggestion
[Like this](images/help/borders.png)
```
This typo should be fixed in the file name itself as well.

Reply:
Don't forget to add the `!` at the start of this line so it's displayed as an image not a link :)

Reply:
I thought i added it 
thanks

Reply:
I don't see a reason to rename the file
> https://github.com/Chatterino/wiki/pull/98#discussion_r684827769

Reply:
> I don't see a reason to rename the file

[Because "boarders" are something completely else](https://translate.google.com/?sl=en&tl=de&text=boarders&op=translate)

Reply:
oh ok will rename